ruby3.2-ffi.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: ruby3.2-ffi
+  version: 1.17.0
+  epoch: 0
+  description: Ruby FFI
+  copyright:
+    - license: BSD-3-Clause And MIT
+  dependencies:
+    runtime:
```

Comment:
These runtime dependencies don't look correct to me. They should just be `ruby-3.2` and `libffi`, both should be added via SCA. We could remove the entire `runtime` section here